.github/workflows/latest-matrix-sdk-crypto.yml

```diff
@@ -0,0 +1,87 @@
+name: Build against latest matrix-sdk-crypto
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '22 1 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  test-against-latest-matrix-sdk-crypto:
+    name: test-against-latest-matrix-sdk-crypto
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: wasm32-unknown-unknown
+
+      - name: Load cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          save-if: ${{ github.ref == 'refs/heads/main' }}
+
+      - name: Update to the latest matrix-sdk-crypto version
+        run: cargo update matrix-sdk-crypto
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.0
```

Comment:
Does it need to be synchronized with other tasks/workflows/documentation? (If so, I try to add comments to *all* the places that need keeping in sync.)

Reply:
Thanks, added in f5ec776